Cubical/HITs/ListedFiniteSet/Base.agda

```diff
@@ -89,3 +91,27 @@ module PropElim {ℓ}
     (λ _ _ _ → isOfHLevel→isOfHLevelDep 1 trunc* _ _ _)
     (λ _ _ → isOfHLevel→isOfHLevelDep 1 trunc* _ _ _)
     λ xs → isProp→isSet (trunc* xs)
+
+_++_ : ∀ (xs ys : LFSet A) → LFSet A
```

Comment:
This should have some fixity, maybe same as for the corresponding list operation?

Reply:
List has `infixr 5 _++_` and that's what I have added.